index.html

```diff
@@ -86,5 +86,12 @@ <h2 class="tech-name">Django</h2>
       </p>
     </div>
   </div>
+  <div class="footer">
+    <h2 class="footer-title">Contact us!</h2>
+    <div class="footer-container">
```

Comment:
since this is a form with an input and a submit this element should be a `form` tag. this allows the type='submit' button to do the form submit.